kotlin-native-tests/src/test/java/DataClassTests.java

```diff
@@ -30,4 +30,11 @@ public void testSimpleDataEquals() {
         SimpleDataClassEquals simpleDataClassEquals = new SimpleDataClassEquals();
         assert (simpleDataClassEquals.main(args) == true);
     }
-}
\ No newline at end of file
+
+// todo copy is called doCopy, but it's not worth fixing for now since it takes all params like the constructor.
+//    @Test
+//    public void testSimpleDataClassCopy() {
+//        SimpleDataClassCopy simpleDataClassCopy = new SimpleDataClassCopy();
+//        assert (simpleDataClassCopy.main(args) == true);
+//    }
+}
```

Comment:
Adding a commented out test?

Reply:
It's kind of our documentation, without doing any documentation.